src/views/student-surveys.vue

```diff
@@ -88,9 +88,16 @@ export default {
                     });
                 }
                 let ret = {
-                    survey_id: data.DATA[0].survey_id,
-                    responses: responses
+                    what: "post_surveys",
+                    survey_instance_id: data.DATA.survey_instance_id,
+                    user_id: user_id,
+                    question_responses: responses
                 };
+
+                fetch("student_survey.php", {
+                    method: "POST",
+                    body: JSON.stringify(ret)
+                }).catch(err => console.log(err));
```

Comment:
`console.warn`

Reply:
I think I'm just going to emit this error to the parent.